Instance .tf

```diff
@@ -0,0 +1,13 @@
+variable "awsprops" {
+  type = "map"
+  default = {
+    region = "us-east-1a"
+    vpc = "vpc-5234832d"
+    ami = "ami-0c1bea58988a989155"
+    itype = "t2.micro"
+    subnet = "subnet-81896c8e"
+    publicip = true
+    keyname = "myseckey"
+    secgroupname = "IAC-Sec-Group"
+  }
+}

```

Comment:
the variable needs to be with no values, so it can be reused for other environments